tests/unit/stream_alert_cli/test_helpers.py

```diff
@@ -95,3 +101,59 @@ def test_get_rules_from_test_events():
             returned_rules = helpers.get_rules_from_test_events('fake/path')
 
         assert_items_equal(rules, returned_rules)
+
+
+def test_record_to_schema_no_recurse():
+    """CLI - Helpers - Record to Schema, Non-recursive"""
+
+    record = {
+        'dict': {
+            'boolean': False,
+            'integer': 123
+        },
+        'list': [],
+        'string': 'this is a string',
+        'float': 1234.56,
+        'integer': 1234,
+        'boolean': True
+    }
+
+    result = helpers.record_to_schema(record, recursive=False)
+
+    assert_is_instance(result['dict'], dict)
+    assert_is_instance(result['list'], list)
+    assert_equal(result['string'], 'string')
+    assert_equal(result['float'], 'float')
+    assert_equal(result['integer'], 'integer')
+    assert_equal(result['boolean'], 'boolean')
+    # Check to make no recursion occurred
+    assert_equal(len(result['dict']), 0)
```

Comment:
Instead of all these different statements, can you have just a single `assert_equal`? I find that easier to read and understand

```python
result = helpers.record_to_schema(record, recursive=False)

expected = {
    'dict': {},
    'list': [],
    'string': 'string',
    ...,
}
assert_equal(result, expected)
```

Also down below